src/components/FilterTabs/FilterTabs.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { Tabs as MuiTabs, TabsProps as MuiTabsProps, makeStyles } from '@material-ui/core';
+import { FilterTab } from '../FilterTab/FilterTab';
+export type TabsProps = MuiTabsProps;
+
+export const FilterTabs = (props: TabsProps): JSX.Element => {
+  const [value, setValue] = useState(1);
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
```

Comment:
Use type from the library.
`const handleChange: MuiTabsProps ['onChange']` = (event, newValue) => {